R/tm_a_regression.R

```diff
@@ -178,30 +178,38 @@ ui_a_regression <- function(id, ...) {
         selected = args$plot_choices[args$default_plot_type]
       ),
       checkboxInput(ns("show_outlier"), label = "Display outlier labels", value = TRUE),
-      shinyjs::hidden(teal.widgets::optionalSliderInput(
-        ns("outlier"),
-        div(
-          class = "teal-tooltip",
-          tagList(
-            "Outlier definition:",
-            icon("circle-info"),
-            span(
-              class = "tooltiptext",
-              paste(
-                "Use the slider to choose the cut-off value to define outliers.",
-                "Points with a Cook's distance greater than",
-                "the value on the slider times the mean of the Cook's distance of the dataset will have labels."
+      conditionalPanel(
+        condition = "input['show_outlier']",
+        ns = ns,
+        teal.widgets::optionalSliderInput(
+          ns("outlier"),
+          div(
+            class = "teal-tooltip",
+            tagList(
+              "Outlier definition:",
+              icon("circle-info"),
+              span(
+                class = "tooltiptext",
+                paste(
+                  "Use the slider to choose the cut-off value to define outliers.",
+                  "Points with a Cook's distance greater than",
+                  "the value on the slider times the mean of the Cook's distance of the dataset will have labels."
+                )
               )
             )
-          )
-        ),
-        min = 1, max = 10, value = 9, ticks = FALSE, step = .1
-      )),
-      shinyjs::hidden(teal.widgets::optionalSelectInput(
-        ns("label_var"),
-        multiple = FALSE,
-        label = "Outlier label"
-      )),
+          ),
+          min = 1, max = 10, value = 9, ticks = FALSE, step = .1
+        )
+      ),
+      conditionalPanel(
+        condition = "input['show_outlier']",
```

Comment:
as this is the same condition, couldn't it be moved into the conditionalPanel above?